src/tools/pub/cmd/pub-global.md

```diff
@@ -110,8 +108,42 @@ and passes in two arguments.
 $ pub global run foo:bar arg1 arg2
 {% endprettify %}
 
+
 ### Running a script from your PATH
 
+In order to run a script directly from your path, it must be specified
+under the `executables` tag in its pubspec file.
+
+For example, activate Stagehand, the Dart project generator:
+
+{% prettify sh %}
+$ pub global activate stagehand
+{% endprettify %}
+
+This command installs the package to the system cache on your machine.
+Because the `stagehand` executable is listed under the `executables` tag
+in the pubspec file, pub creates a corresponding shell script in the `bin`
+directory of the system cache.
+
+If you haven't already done so, add the bin directory for the
+[system cache](/tools/pub/glossary#system-cache)
+(maintained by the [`pub cache`](/tools/pub/cmd/pub-cache) command),
+to your path. On Mac and Linux, add `~/.pub-cache/bin` to your path.
+On Windows, add `C:\Users\<user name>\AppData\Roaming\Pub\Cache\bin`
```

Comment:
I asked Nicholas to check on a Windows machine and he confirmed that location on Windows 10. Do you think it might be different for different versions of Windows? (I think I said Nathan in a comment, but it was Nicholas.)